docs/RecommendedChanges/System/Use AssemblyBuilder.DefineDynamicAssembly.md

```diff
@@ -0,0 +1,5 @@
+### Recommended Action
+Use AssemblyBuilder.DefineDynamicAssembly
+
+### Affected APIs
+* `M:System.AppDomain.DefineDynamicAssembly`
```

Comment:
@jkotas did you verify that this works? Doc IDs are usually matched by `==` and you probably to apply this to all overloads, right?

Reply:
@conniey?

Reply:
@terrajobst good catch. I'll create a PR to fix these.